README.md

```diff
@@ -49,48 +69,6 @@ which gets published
 The code that is published to JSBin can be found in
 [content/code/](content/code/).
 
-## Forking and running locally
```

Comment:
The link above still tries to point to this:

**This configuration is not required for basic documentation creation and modification, so most
editors can simply skip to
[Forking and running locally](#forking-and-running-locally).**